### ISSUE TITLE
Unusual parameter parsing

### ISSUE THREAD

**Issue by lbthomsen** [subutai-io/agent#202]
The "standard" UNIX way of handling parameters is to allow short parameters to be bundled together:

`triton418:~# subutai list -cp
Incorrect Usage: flag provided but not defined: -cp

NAME:
   subutai list - list Subutai container

USAGE:
   subutai list [command options] [arguments...]

OPTIONS:
   --container, -c  containers only
   --template, -t   templates only
   --info, -i       detailed container info
   --ancestor, -a   with ancestors
   --parent, -p     with parent
   
triton418:~# subutai list -c -p
CONTAINER       PARENT
---------       ------
container1      ubuntu16`

so the following should all do the same:

subutai list -cp
subutai list -c -p
subutai list --container --parent

**Comment by Dilshat** [comment]
Yes, currently standard Go flags are used which is a legacy of Plan 9 system.
As a future enhancement we should use GNU style flags

**Comment by crioto** [comment]
@Dilshat this is incorrect. Agent uses github.com/urfave/cli library to parse flags which supports options combination out of the box https://github.com/urfave/cli#combining-short-bool-options

**Comment by Dilshat** [comment]
@crioto yes, I don't mean implementation part, I mean interface: that currently agent does not support combined flags

**Comment by Dilshat** [comment]
We need to support short long and combined flag eventually.

**Comment by Dilshat** [comment]
How to test:

run command `subutai update rh -cd`
it should check for update as well as print debug level logs

**Comment by Shahrizada** [comment]
Test case: 

```95E25671040BCC5FBFCB8228E6A00E4D4DB16602@/:>subutai update rh -cd
  time="2018-08-29 03:38:36" level=debug msg="Init lock subutai.update.rh" 
time="2018-08-29 03:38:36" level=debug msg="Locking file subutai.update.rh" 
time="2018-08-29 03:38:39" level=debug msg="Updating apt index " 
time="2018-08-29 03:38:40" level=debug msg="Checking for available update Inst openssh-sftp-server [1:7.4p1-10+deb9u3] (1:7.4p1-10+deb9u4 Debian-Security:9/stable [amd64]) []
Inst openssh-server [1:7.4p1-10+deb9u3] (1:7.4p1-10+deb9u4 Debian-Security:9/stable [amd64]) []
Inst openssh-client [1:7.4p1-10+deb9u3] (1:7.4p1-10+deb9u4 Debian-Security:9/stable [amd64])
Inst linux-compiler-gcc-6-x86 [4.9.110-1] (4.9.110-3+deb9u4 Debian-Security:9/stable [amd64])
Inst linux-headers-4.9.0-7-amd64 [4.9.110-1] (4.9.110-3+deb9u2 Debian-Security:9/stable [amd64]) []
Inst linux-headers-4.9.0-7-common [4.9.110-1] (4.9.110-3+deb9u2 Debian-Security:9/stable [all])
Inst linux-kbuild-4.9 [4.9.110-1] (4.9.110-3+deb9u4 Debian-Security:9/stable [amd64])
Inst linux-headers-4.9.0-8-common (4.9.110-3+deb9u4 Debian-Security:9/stable [all])
Inst linux-headers-4.9.0-8-amd64 (4.9.110-3+deb9u4 Debian-Security:9/stable [amd64])
Inst linux-headers-amd64 [4.9+80+deb9u5] (4.9+80+deb9u6 Debian-Security:9/stable [amd64])
Inst linux-image-4.9.0-7-amd64 [4.9.110-1] (4.9.110-3+deb9u2 Debian-Security:9/stable [amd64])
Inst linux-image-4.9.0-8-amd64 (4.9.110-3+deb9u4 Debian-Security:9/stable [amd64])
Inst linux-image-amd64 [4.9+80+deb9u5] (4.9+80+deb9u6 Debian-Security:9/stable [amd64])
Inst linux-libc-dev [4.9.110-1] (4.9.110-3+deb9u4 Debian-Security:9/stable [amd64])
Conf openssh-sftp-server (1:7.4p1-10+deb9u4 Debian-Security:9/stable [amd64])
Conf openssh-server (1:7.4p1-10+deb9u4 Debian-Security:9/stable [amd64])
Conf openssh-client (1:7.4p1-10+deb9u4 Debian-Security:9/stable [amd64])
Conf linux-compiler-gcc-6-x86 (4.9.110-3+deb9u4 Debian-Security:9/stable [amd64])
Conf linux-headers-4.9.0-7-amd64 (4.9.110-3+deb9u2 Debian-Security:9/stable [amd64])
Conf linux-headers-4.9.0-7-common (4.9.110-3+deb9u2 Debian-Security:9/stable [all])
Conf linux-kbuild-4.9 (4.9.110-3+deb9u4 Debian-Security:9/stable [amd64])
Conf linux-headers-4.9.0-8-common (4.9.110-3+deb9u4 Debian-Security:9/stable [all])
Conf linux-headers-4.9.0-8-amd64 (4.9.110-3+deb9u4 Debian-Security:9/stable [amd64])
Conf linux-headers-amd64 (4.9+80+deb9u6 Debian-Security:9/stable [amd64])
Conf linux-image-4.9.0-7-amd64 (4.9.110-3+deb9u2 Debian-Security:9/stable [amd64])
Conf linux-image-4.9.0-8-amd64 (4.9.110-3+deb9u4 Debian-Security:9/stable [amd64])
Conf linux-image-amd64 (4.9+80+deb9u6 Debian-Security:9/stable [amd64])
Conf linux-libc-dev (4.9.110-3+deb9u4 Debian-Security:9/stable [amd64])
W: --force-yes is deprecated, use one of the options starting with --allow instead.
" 
time="2018-08-29 03:38:40" level=info msg="Update is available" 

95E25671040BCC5FBFCB8228E6A00E4D4DB16602@/:>```
